tg/views.py

```diff
@@ -161,9 +161,12 @@ def _unread_color(self, is_selected: bool = False) -> int:
             return color | reverse
         return color
 
-    def _chat_attributes(self, is_selected: bool = False) -> Tuple[int, ...]:
+    def _chat_attributes(
+        self, is_selected: bool, title: str
+    ) -> Tuple[int, ...]:
         attrs = (
             get_color(cyan, -1),
+            get_color(get_color_by_user(title), -1),
```

Comment:
this function should be renamed to more generic name e.g `get_color_by_text` | `get_color_from_str`